build/common.sh

```diff
@@ -55,7 +43,16 @@ readonly KUBE_BUILD_PPC64LE="${KUBE_BUILD_PPC64LE:-n}"
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
 readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat ${KUBE_ROOT}/build/build-image/cross/VERSION)"
-# KUBE_BUILD_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+# KUBE_DATA_CONTAINER_NAME=kube-build-data-<hash>"
```

Comment:
Yeah - this is left over.  I was documenting the constants up here but some of them aren't really constant and are set in `kube::build::verify_prereqs` now.  There is enough stuff there that it should be cool deleting this comment.